src/components/commons/ReviewCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
```suggestion
import { Fragment } from 'react';
```

이건 그냥 제안인데요, 임포트 할때부터 Fragment를 뽑아와서
밑에서 사용할땐 `<React.Fragment>` 가 아닌 `<Fragment>` 라고만 써도 되도록 해도
좋을거 같습니다!

Reply:
오 맞네요!! 수정할게요! 감사합니다~